lambdas/service/app.py

```diff
@@ -783,6 +780,21 @@ def validate_order(order: str):
         raise BRE(f'Unknown order `{order}`. Must be one of {supported_orders}')
 
 
+def validate_search_before(search_before: str):
+    validate_json_param('search_before', search_before)
+
+
+def validate_search_after(search_after: str):
+    validate_json_param('search_after', search_after)
+
+
+def validate_json_param(name: str, json_: str) -> MutableJSON:
+    try:
+        return json.loads(json_)
+    except json.decoder.JSONDecodeError:
+        raise BRE(f'The `{name}` parameter is not valid JSON')
```

Comment:
```suggestion
        raise BRE(f'The {name!r} parameter is not valid JSON')
```

We're moving away from backquotes and there is precedent in this module that shows that we already started, so I'm OK with having a mix of single quotes and backquotes.